jib-core/src/test/java/com/google/cloud/tools/jib/cache/CacheStorageWriterTest.java

```diff
@@ -336,6 +340,23 @@ public void testWriteLocalConfig() throws IOException, URISyntaxException, Diges
     Assert.assertEquals("wasm", savedContainerConfig.getArchitecture());
   }
 
+  @Test
+  public void testMoveIfDoesNotExist_exceptionAfterFailure() {
+    Exception exception =
+        assertThrows(
+            IOException.class,
+            () -> CacheStorageWriter.moveIfDoesNotExist(Paths.get("/foo"), Paths.get("/bar")));
+    assertThat(exception)
+        .hasMessageThat()
+        .isEqualTo(
+            "unable to move: /foo to /bar; such failures are often caused by interference from "
+                + "antivirus (https://github.com/GoogleContainerTools/jib/issues/3127#issuecomment-796838294), "
+                + "or rarely if the operation is not supported by the file system (for example: "
+                + "special non-local file system)");
```

Comment:
sometimes asserting on the whole message can make the test brittle. How about we assert on just the essential part? : `unable to move: /foo to /bar; such failures are often caused by interference from antivirus`